starknet_devnet/fee_token.py

```diff
@@ -26,10 +27,12 @@ class FeeToken:
     # HASH = to_bytes(compute_class_hash(contract_class=FeeToken.get_contract_class()))
     HASH = 0x6A22BF63C7BC07EFFA39A25DFBD21523D211DB0100A0AFD054D172B81840EAF
     HASH_BYTES = to_bytes(HASH)
+    class_hash_bytes = HASH_BYTES
 
     # Taken from
     # https://github.com/starknet-community-libs/starknet-addresses/blob/df19b17d2c83f11c30e65e2373e8a0c65446f17c/bridged_tokens/goerli.json
     ADDRESS = 0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7
+    address = ADDRESS
```

Comment:
Seems redundant to have both `ADDRESS` and `address` (same for hash bytes). Can you make it so that we have just one?

Reply:
That's why I did it the way it was before, to avoid redundancy.

Do you suggest we remove `ADDRESS` in favor of `address` and replace every use of it in the code base ?

Reply:
Can PredeployedAccount parent class can depend on `ADDRESS` ?

Reply:
That'd make it look like a constant which isn't in the case of `Account`

Reply:
Ok, good point. In that case there should be an **abstract** getter: either something like `get_address` or a formally annotated property (can the latter even be abstract?). Then all child classes other than `Account` should implement it by returning their respective precalculated constants and `Account` can return `self.address` (or it should be `self._address`).

Reply:
We can keep the predeployed class as it is and replace
```python
address = ADDRESS 
```
With
```python
@property
def address(self) -> int:
    return self.ADRRESS
```

Reply:
Looks good!

Reply:
I think it's ok.

(It is kind of a compromise between the `@property` approach and having `address` and `ADDRESS` properties next to each other, but it's reasonable IMO)